### DISCUSSION TITLE
Organização da documentação

### DISCUSSION THREAD

**Discussion by kaioenzo** [cerejeiros/mobile#22] [Polls]
Como forma de organizar e achar a melhor forma para a documentação do projeto é interessante que discutamos onde a documentação deve ser alocada e como deve ser atualizada.
Eu sugiro que não haja uma branch específica para a documentação, pois, acredito que quando realizarmos as tarefas já deveríamos na branch que criamos algo, escrever a documentação daquilo, se necessário. Ao documentar na mesma branch em que as tarefas são realizadas, evita-se a necessidade de manter várias branches separadas para a documentação, o que pode tornar o processo de documentação mais complicado e confuso. Além disso, manter a documentação junto com o código também ajuda a garantir que o histórico do projeto seja mantido e que as informações estejam sempre atualizadas. Em resumo, documentar na mesma branch em que as tarefas são realizadas é uma prática recomendada, desde que a equipe defina um padrão claro e siga as boas práticas de documentação, como revisão regular e atualização das informações.
@cerejeiros/back-end @cerejeiros/design @cerejeiros/front-end @cerejeiros/testers

**Comment by BeyondMagic** [comment]
O _Pull Request_ #16 (agora #20) foi criado apenas para adicionar log de reuniões (como foi discutido na segunda reunião que tivemos), já que é algo que todos individualmente devem contribuir, não demanda muito, e dicutimos tudo.
O que vejo também de benefício uma fixação dos comandos básicos de _git_.

Eu também concordo que documentação de **qualquer outra coisa** deve ser feito em seus próprios PRs e branches, como todo e qualquer projeto.

___

#20 será refixada a mensagem de que o arquivo escolhido */docs/documentation.md* é para reuniões !
